ui/src/config/section/infra/routers.js

```diff
@@ -33,6 +33,10 @@ export default {
   searchFilters: ['name', 'zoneid', 'podid', 'clusterid'],
   details: ['name', 'id', 'version', 'softwareversion', 'requiresupgrade', 'guestnetworkname', 'vpcname', 'publicip', 'guestipaddress', 'linklocalip', 'serviceofferingname', 'networkdomain', 'isredundantrouter', 'redundantstate', 'hostname', 'account', 'zonename', 'created', 'hostcontrolstate'],
   resourceType: 'VirtualRouter',
+  filters: () => {
+    const filters = ['running', 'stopped']
```

Comment:
@vishesh92 could you explore other states for all these resources ? (Including some useful transition states like migrating)